packages/opensrp-plans/src/components/ListView/tests/index.test.tsx

```diff
@@ -65,6 +66,111 @@ describe('List view Page', () => {
       `"Active Missions + New MissionNameDateActionsNo Data"`
     );
   });
+  it('renders Draft Missions Title', async () => {
+    fetch.mockResponse(JSON.stringify([]));
```

Comment:
should we mock some data with the status being matched and assert that it shows  up on the page?